boards/arm/colibri_imx7d_m4/board.h

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2018, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __INC_BOARD_H
+#define __INC_BOARD_H
+
+#include <soc.h>
+
+/* The board name */
+#define BOARD_NAME                            "IMX7_COLIBRI_M4"
+#define BOARD_DOMAIN_ID                       (1)
```

Comment:
I think DOMAIN_ID should be a CONFIG param in the SoC and default to 1.  Than board code can change the default in Kconfig.defconfig if desired.